File: 프로그래머스/lv0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.js
function solution(my_str, n) {
    var answer = [];
    let word = ''
    for (let i of my_str) {
        if (word.length === n) {
            answer.push(word)
            word = i
        }
        else {
            word += i
        }
        
    }
    answer.push(word)
    return answer;
}

File: 프로그래머스/lv0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.py
def solution(my_str, n):
    answer = []
    my_str = list(my_str)
    
    n_word = ''
    for alpha in my_str:
        n_word += alpha
        if len(n_word) == n:
            answer.append(n_word)
            n_word = ''
        
    answer.append(n_word)
    if len(answer[-1]) == 0:
        answer.pop()
    return answer